src/plugin/ipc/socket/socketconnection.h

```diff
@@ -119,23 +137,23 @@ namespace dmtcp
       virtual void serializeSubClass(jalib::JBinarySerializer& o);
     private:
       TcpConnection& asTcp();
-    private:
-      int32_t                   _listenBacklog;
-      union {
-        socklen_t               _bindAddrlen;
-        socklen_t               _connectAddrlen;
-      };
-      union {
-        /* See 'man socket.h' or POSIX for 'struct sockaddr_storage' */
-        struct sockaddr_storage _bindAddr;
-        struct sockaddr_storage _connectAddr;
-      };
-      ConnectionIdentifier    _remotePeerId;
   };
 
   class RawSocketConnection : public Connection, public SocketConnection
   {
     public:
+      enum RawType
+      {
+        RAW_INVALID = RAW,
+        RAW_ERROR,
+        RAW_CREATED,
```

Comment:
Can we find a more meaningful name for `RAW_CREATED`?  Does this mean raw socket that is uninitialized?  How about `RAW_UNINITIALIZED` or `RAW_UNBOUND` or `RAW_NEW_ENDPOINT`?